docs/source-fabric/advanced/compile.rst

```diff
@@ -275,11 +275,11 @@ In the future, this option will become the default.
     # Compile the model
     model = torch.compile(model)
 
-    # Default: `fabric.setup()` will not reapply the compilation over DDP/FSDP
-    model = fabric.setup(model, _reapply_compile=False)
+    # Default: `fabric.setup()` will configure compilation over DDP/FSDP for you
+    model = fabric.setup(model, reapply_compile=True)
```

Comment:
Should we keep the argument protected? We don't want users to think they need to touch this. It should only be an emergency escape hatch

Reply:
That's also fine with me. Would you then still keep the docs section as it is right now?

Reply:
Sure, maybe move it to the bottom